### ISSUE TITLE
mod for VR?

### ISSUE THREAD

**Issue by richard987** [Nukem9/dlssg-to-fsr3#383]
Nice mod... will there be a version that does dlss3/frame gen for steamVR, eg microsoft flight simulator in VR mode?

**Comment by TimmyP777** [comment]
No. Framegen wont work in VR. Both are just iterations on ASW and MR that has been used in VR for years. The "updated" OFA chip nvidia claimed was necessary for DLSS3, was actually originally developed for VR as another alternative to the other methods.

In other words, VR is already doing it.

**Comment by maartenvdezz** [comment]
yeah for MSFS 24 it works perfect in desktop mode but in VR there is no toggle!
